package.json

```diff
@@ -6,26 +6,53 @@
   "private": true,
   "scripts": {
     "dev": "cross-env NODE_ENV=development webpack-dev-server --open --hot",
-    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules"
+    "build": "cross-env NODE_ENV=production webpack --progress --hide-modules",
+    "test": "jest",
+    "lint": "./node_modules/.bin/eslint . ./**/*.vue"
```

Comment:
we followed this tutorial for setup: https://eslint.org/docs/user-guide/getting-started
we only have the linter as a node module

Reply:
Yes, but do you see the difference?

If you run
```bash
$ somecommand
```

Then your shell will look in all those locations where it may find executables. Here is the content of my `$PATH` variable:

```sh
echo $PATH
/home/robert/go/bin /usr/share/neo4j/bin/ /opt/android-sdk/platform-tools /home/robert/.fzf/bin /home/robert/go/bin /usr/share/neo4j/bin/ /opt/android-sdk/platform-tools /home/robert/go/bin /usr/share/neo4j/bin/ /opt/android-sdk/platform-tools /usr/local/sbin /usr/local/bin /usr/bin /usr/bin/site_perl /usr/bin/vendor_perl /usr/bin/core_perl
```

But if you run

```sh
npm run somecommand
```

It will first look into `node_modules/.bin` and *then* it will resolve the executable via $PATH. That's why it's not necessary to hard-code the path in your `package.json`. But if you would run it without `npm run` then you have to specify the path to the executable.

In your tutorial the author explains how you can call `eslint` *without* a script in `package.json`.